src/java.base/share/conf/security/java.security

```diff
@@ -28,6 +28,46 @@
 # Properties in this file are typically parsed only once. If any of the
 # properties are modified, applications should be restarted to ensure the
 # changes are properly reflected.
+#
+# The special "include" property can be defined one or multiple times in
+# this file with a filesystem path value. The effect of each definition
+# is to include a referred security properties file inline, adding all
+# its properties. Security properties defined before an include statement
+# may be overridden by properties in the included file, if their names
+# match. Conversely, properties defined after an include statement may
+# override properties in the included file.
+#
+# Included files, as well as files pointed by java.security.properties,
+# can include other files recursively. Paths may be absolute or relative.
+# Each relative path is resolved against the base file containing its
+# "include" definition, if local. Paths may contain system properties for
+# expansion in the form of ${system.property}. If a system property does
+# not have a value, it expands to the empty string.
+#
+# System properties in included paths provide flexibility for users to
+# switch between predefined security profiles on a per-execution basis.
+# Profile selection may be required or optional depending on the files
+# layout, as shown in example #3 below.
+#
+# An error will be thrown if a file cannot be included, either because it
+# does not exist, cannot be resolved, or is recursively included more than
+# once. For a secure JDK configuration, it is important to check OS
+# permissions assigned to any file included.
+#
+# Examples:
+#   1) include ${java.home}/conf/security/extra.security
+#   2) include extra.security
+#   3) include ${java.home}/conf/security/profile${SecurityProfile}.security
+#
+# In example #3, if Java is launched with the argument -DSecurityProfile=FIPS,
+# the profile defined in profileFIPS.security is applied. An error is thrown
+# if this file is not found. Other files may exist in the same directory,
+# enabling the selection of alternative profiles. If SecurityProfile is not
+# set, it expands to the empty string. When a profile.security file exists
+# (may be empty), it defines a default security profile. When it does not,
+# an error is thrown and explicit profile selection is required. In order
+# to override properties defined in this file, the include statement may be
+# placed at the end.
```

Comment:
Same with this paragraph. I think it is ok/useful to include the 3 examples but explaining how one might deploy the include files with FIPS as an example is more for a programming guide. Also, some text is repeated - you already indicated an error is thrown if a file does not exist and the property defaults to the empty string.

The one sentence I found useful is the last one - but I think you could move this as the last sentence of the first paragraph and it would have more impact. I would change "may" to "should".

Reply:
We completely agree with removing the description of this particular case from the _java.security_ file for the same reasons as before. In regards to the last line, we would remove it as well. It's explained in the first paragraph that included files are _inlined_ into a security properties file. The paragraph continues with a description of what _inlining_ means. The comment about placing the include statement at the end is a particular configuration and can be logically inferred from what _inlining_ is and what the user goals are. I would abstain from recommending particular configurations and let the user make their own analysis and choice. It may be the case that some properties should not be available for profiles and will then be intentionally overwritten in the _java.security_ file after the include statement.

Reply:
That's fine, I think your rationale to not include it makes sense. Please also update the CSR with the text that has been removed and I will add my name as Reviewer.